Generals/Code/GameEngine/Source/GameLogic/Object/Object.cpp

```diff
@@ -2789,6 +2789,11 @@ Bool Object::hasSpecialPower( SpecialPowerType type ) const
 //-------------------------------------------------------------------------------------------------
 void Object::onVeterancyLevelChanged( VeterancyLevel oldLevel, VeterancyLevel newLevel, Bool provideFeedback )
 {
+#if !RETAIL_COMPATIBLE_CRC
+	if (isEffectivelyDead())
```

Comment:
Can add TheSuperHackers comment because it is user facing, even if obvious what it does.